main.cpp

```diff
@@ -166,10 +194,26 @@ struct Computer
     
     void playMusic(int howLoud = 11); //play it loud!
     
-    void displayNumbers(int numberCount);
+    void displayVolume(int loudness = 1);
 };
 
-
+void Computer::runQuickbooks(bool isQBInstalled)
+{
+    isQBInstalled = false;
+}
+void Computer::displayVolume(int loudness)
+{
+    loudness = 11;
+}
+void Computer::playMusic(int howLoud)
+{
+    int loudness;
+    if (howLoud == 11)
+    {
+        loudness = howLoud;
+    }
+     
+}
```

Comment:
if `howLoud` is not 11, what does `loudness` get set to?   
- [ ] revise.  Always initialize your variables.